### PR TITLE
Update to using Microsoft.Taef Nuget package

### PR DESCRIPTION
**Pull request by kythant**
Stop using the old manually named Microsoft.Taef.Redist.Wlk package and start using the updated newly renamed Microsoft.Taef package from the ES feed.

### CODE REVIEW

File: build/Helix/NuGet.config [hunk 1]
```diff
@@ -19,7 +19,7 @@
     <clear />
     <add key="NuGet official package source" value="https://nuget.org/api/v3/" />
     <add key="dotnetfeed" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
-    <add key="ProjectReunion-Dependencies" value="https://pkgs.dev.azure.com/ms/ProjectReunion/_packaging/ProjectReunion-Dependencies/nuget/v3/index.json" />
+    <add key="WindowsES-External" value="https://microsoft.pkgs.visualstudio.com/_packaging/WindowsES-External/nuget/v3/index.json" />
```
Comment (EHO-makai) [inline build/Helix/NuGet.config:22]: why this change?

File: nuget.config [hunk 2]
```diff
@@ -1,13 +1,10 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
-  <disabledPackageSources>
```
Comment (EHO-makai) [inline nuget.config:3]: wasn't this required a little while ago?

File: build/Helix/NuGet.config (hunk 1 above)
Comment (kythant) [reply build/Helix/NuGet.config:22]: I renamed the package have Microsoft. in the front and I added it to ProjectReunion-Dependencies.
Now we can unlist it from there and pull it from WindowsES-External instead.

File: nuget.config (hunk 2 above)
Comment (kythant) [reply nuget.config:3]: Yea it was in two different places so I am just removing the extra one.

File: nuget.config [hunk 3]
```diff
@@ -1,13 +1,10 @@
...
-    <clear />
-  </disabledPackageSources>
-
   <packageSources>
     <clear />
     <add key="NuGet" value="https://api.nuget.org/v3/index.json" />
     <add key="ProjectReunion-Dependencies" value="https://pkgs.dev.azure.com/ms/ProjectReunion/_packaging/ProjectReunion-Dependencies/nuget/v3/index.json" />
+    <add key="WindowsES-External" value="https://microsoft.pkgs.visualstudio.com/_packaging/WindowsES-External/nuget/v3/index.json" />	
   </packageSources>
   <disabledPackageSources>
```
Comment (kythant) [inline nuget.config:9]: Here

**Review by DrusTheAxe — APPROVED** [review]

File: dev/MRTCore/build-mrt.yml [hunk 4]
```diff
@@ -0,0 +1,191 @@
+# Universal Windows Platform
```
Comment (EHO-makai) [inline dev/MRTCore/build-mrt.yml:1]: is this a move?

File: dev/MRTCore/publish-mrt.yml [hunk 5]
```diff
@@ -0,0 +1,198 @@
+parameters:
```
Comment (EHO-makai) [inline dev/MRTCore/publish-mrt.yml:1]: is this a move?

File: dev/MRTCore/build-mrt.yml (hunk 4 above)
Comment (kythant) [reply dev/MRTCore/build-mrt.yml:1]: Yup!

File: dev/MRTCore/publish-mrt.yml (hunk 5 above)
Comment (kythant) [reply dev/MRTCore/publish-mrt.yml:1]: Move and a split

File: build/ProjectReunion-CI.yml [hunk 6]
```diff
@@ -38,11 +38,38 @@ jobs:
...
+      Release_x64:
+        buildPlatform: 'x64'
+        buildConfiguration: 'Release'
+      Release_Arm64:
+        buildPlatform: 'ARM64'
+        buildConfiguration: 'Release'
+  steps:
+  - task: NuGetAuthenticate@0
+    inputs:
+      nuGetServiceConnections: 'WindowsES'
```
Comment (huichen123) [inline build/ProjectReunion-CI.yml:59]: why is this not needed in ProjectReunion-BuildFoundation.yml?

Comment (kythant) [reply build/ProjectReunion-CI.yml:59]: It's because calling NuGetAuthenticate on the internal pipeline will cause nuget restore to fail. I don't understand why.

File: nuget.config (hunk 2 above)
Comment (huichen123) [reply nuget.config:3]: i think clear needs to be on top

---
In reply to: [581489335](https://github.com/microsoft/ProjectReunion/pull/473#discussion_r581489335) [](ancestors = 581489335)

File: build/ProjectReunion-CI.yml (hunk 6 above)
Comment (huichen123) [reply build/ProjectReunion-CI.yml:59]: probably worth a comment

---
In reply to: [581568235](https://github.com/microsoft/ProjectReunion/pull/473#discussion_r581568235) [](ancestors = 581568235)

File: nuget.config (hunk 2 above)
Comment (kythant) [reply nuget.config:3]: https://www.1eswiki.com/wiki/How_to_Securely_Configure_Package_Source_Files

hmm in the example, it's at the bottom

Comment (huichen123) [reply nuget.config:3]: never mind. i was talking about clear at line 4.

---
In reply to: [581569746](https://github.com/microsoft/ProjectReunion/pull/473#discussion_r581569746) [](ancestors = 581569746)

**Review by huichen123 — APPROVED** [review]

**Review by EHO-makai — APPROVED** [review]
Thanks!

File: build/ProjectReunion-CI.yml (hunk 6 above)
Comment (kythant) [reply build/ProjectReunion-CI.yml:59]: Will add a comment later
